R/doc.R

```diff
@@ -47,14 +46,12 @@ dldoc <- function # Make directlabels documentation
   makerd <- function # Make Rd positioning method description
   (L
    ## List of posfuns and plots to match up
-   ){
+  ){
```

Comment:
please avoid deleting white space when it is not relevant

Reply:
Sure, I will revert these changes in the next few minutes

Reply:
@tdhock please review/check again now